draft-ietf-quic-recovery.md

```diff
@@ -1368,7 +1377,14 @@ acked_packet from sent_packets.
        return
      if (congestion_window < ssthresh):
        // Slow start.
-       congestion_window += acked_packet.size
+       if (pacing):
+         congestion_window += acked_packet.size
+       else:
+         // If pacing is not used, limit the increase to
+         // 2*max_datagram_size per ACK as described in {{?RFC3465}}
+         if (cwnd_increase < 2 * max_datagram_size):
+           cwnd_increase += bytes_acked
```

Comment:
cwnd_increase doesn't appear to be used anywhere to change cwnd/etc.  Are more changes necessary?

Reply:
It is only used to limit the increase to 2*MSS in slow start

Reply:
`bytes_acked` doesn't appear anywhere. I think it should be `acked_packet.size`?

Reply:
oops, that's correct. Fixing it